src/interfaces/modular/SGBase.i

```diff
@@ -107,6 +108,7 @@ public void readExternal(java.io.ObjectInput in) throws java.io.IOException, jav
  #include <shogun/base/Version.h>
  #include <shogun/base/Parallel.h>
  #include <shogun/base/SGObject.h>
+ #include <../tests/unit/base/MockObject.h> // Temporary include to test Tags
```

Comment:
Well, we are not going to keep `CMockObject` in shogun forever. Once shogun classes start using tags-parameter framework, we can remove this. But I think that will take some time, if you want I can remove the "temporary comment.